tokio/src/lib.rs

```diff
@@ -66,13 +66,13 @@
 //! is possible to just enable certain features over others. By default, Tokio
 //! does not enable any features but allows one to enable a subset for their use
 //! case. Below is a list of the available feature flags. You may also notice
-//! above each function, struct and trait there is a set of feature flags
-//! that are required for that item to be enabled. If you are new to Tokio it is
-//! recommended that you use the `full` feature flag which will enable everything.
+//! above each function, struct and trait there is listed one or more feature flags
+//! that are required for that item to be used. If you are new to Tokio it is
+//! recommended that you use the `full` feature flag which will enable all public APIs.
```

Comment:
Some of these changes seem unrelated?

Reply:
The new `parking_lot` feature is not included in `full` so I needed to distinguish public API enabling features from internal features (this being the first).  So saying full enables "everything" doesn't work so well anymore, could be confusing.